sessions/repo-cleanup-sprint.md

```diff
@@ -0,0 +1,18 @@
+# Repo Cleanup Sprint
+
+**Description:** There are several issues where we’ve worked out some concrete ways we can improve the understandability and organization of the plethora of repos we have on GitHub, but we have failed to actually execute on them for each repo. Let’s have project owners (who have the rights to make changes to the description of a repo or archive a repo) sit down and just *do that work.* Since folks will be sitting down and working together, it gives everyone a good opportunity to quickly gut-check each other on things like whether a description is good or it makes sense to archive a repo.
+
+- [Develop a standard way to indicate a repo/project is deprecated (ipfs/docs#54)](https://github.com/ipfs/docs/issues/54)
+- [Make sure every repo has a description and readme (ipfs/docs#55)](https://github.com/ipfs/docs/issues/55)
+- [Develop a standard way to indicate non-code repos (ipfs/docs#57)](https://github.com/ipfs/docs/issues/57)
+- Make sure repos adhere to the [licensing policy](https://github.com/ipfs/community/blob/68f2fc02c4384eeb765ebc7547ea07f3aa2268c1/docs/licensing-policy.md), now that we have one :)
+
+**Activity:** Work sprint.
```

Comment:
Can we have a randomizer distributing X number of repos by each participant? It will get people to look at repos that they never looked before and have that new user perspective, leading to understand better the pain points. Then they can submit a PR for the deprecation and the admin can merge/make the final changes.

Reply:
> Then they can submit a PR for the deprecation and the admin can merge/make the final changes.

If not clear from the description, my intent here is really to lock repo owners in a room for a few minutes until they do this. We could have other people make issues, [but](https://github.com/ipfs/js-datastore-core/issues/11) [I’ve](https://github.com/ipfs/go-ipfs-cmdkit/issues/20) [been](https://github.com/ipfs/go-ds-badger/issues/16) [doing](https://github.com/ipfs/go-block-format/issues/10) [that](https://github.com/ipfs/kubernetes-ipfs/issues/50) for a while and they aren’t getting merged (those are all linked from the issues directly linked in the description).

Reply:
Anyway, I guess what I’m saying is: having folks who don’t own these repos in this session could help scale it up slightly, but it feels to me like the real bottleneck is repo owners finding the time to just sit down and respond-to/implement/merge the proposals from those other folks.

My thought here was more like “let’s create a dedicated, short time block for those repo owners to focus on just doing that.”